platforms/shared/charts/letsencrypt-issuer/values.yaml

```diff
@@ -0,0 +1,9 @@
+##############################################################################################
+#  Copyright Accenture. All Rights Reserved.
+#
+#  SPDX-License-Identifier: Apache-2.0
+##############################################################################################
+componentns: ""
```

Comment:
We need to mention the example values for these key-value pairs (as one can use these charts manually as well).
For reference: https://github.com/hyperledger/bevel/blob/b7c4459ccd15a23e0d6fa34ec2162167f145dbce/platforms/hyperledger-besu/charts/node_besu/values.yaml#L13

Also its better to call this key as `namespace` instead of componentns (componentns is a terminology coming from the ansible playbook, but here, as one can use the chart manually, calling it componentns is confusing)

Reply:
fixed in pr #2187 as unwanted files got added while squashing